docs/source/tensorbay_cli/getting_start_with_CLI.rst

```diff
@@ -6,6 +6,7 @@ The TensorBay Command Line Interface is a tool to operate on your datasets.
 It supports Windows, Linux, and Mac platforms.
```

Comment:
the file name: `tensorbay_cli/getting_start_with_CLI.rst`
Why `cli` in `tensorbay_cli/` is lowercase, but uppercase in `getting_start_with_CLI.rst`